core/imageroot/var/lib/nethserver/cluster/actions/get-cluster-status/50read

```diff
@@ -28,14 +28,14 @@ import socket
 import hashlib
 import subprocess
 import agent.tasks
-
-def get_ip(host):
-    ip_list = socket.gethostbyaddr(host)[2]
-    return ip_list[0]
+import os
 
 rdb = agent.redis_connect(privileged=True)
 
 cluster_ui_name = rdb.get(f"cluster/ui_name") or ""
+leader_id = int(rdb.hget("cluster/environment", "NODE_ID"))
```

Comment:
Is this the "official" way to find the leader?
If so, maybe we need to document it somewhere.